src/stop-sqm

```diff
@@ -15,6 +15,13 @@
 . ${SQM_LIB_DIR}/defaults.sh
 
 check_state_dir
+# log file for the most recent sqm instance stop
+if [ "$SQM_DEBUG" -eq "1" ] ; then
+    SQM_DEBUG_LOG=${SQM_STOP_LOG}
+    OUTPUT_TARGET=${SQM_DEBUG_LOG}
+    echo "$(date): stop-sqm: Log for interface ${IFACE}"  > ${OUTPUT_TARGET}
+fi
+
```

Comment:
Missing quotes.

Also, better to put the datestamp at the end of the line, so it always starts with the same string (for grepping)

Reply:
Wll fix, thanks.

Reply:
Fixed, see above.